velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -689,30 +694,34 @@ TEST_F(DateTimeFunctionsTest, monthDate) {
 }
 
 TEST_F(DateTimeFunctionsTest, monthTimestampWithTimezone) {
-  EXPECT_EQ(
-      12, evaluateWithTimestampWithTimezone<int64_t>("month(c0)", 0, "-01:00"));
-  EXPECT_EQ(
-      1, evaluateWithTimestampWithTimezone<int64_t>("month(c0)", 0, "+00:00"));
+  const auto monthTimestampWithTimezone =
+      [&](std::optional<TimestampWithTimezone> timestampWithTimezone) {
+        return evaluateOnce<int64_t>(
+            "month(c0)",
+            TIMESTAMP_WITH_TIME_ZONE(),
+            TimestampWithTimezone::pack(timestampWithTimezone));
+      };
+  EXPECT_EQ(12, monthTimestampWithTimezone(TimestampWithTimezone(0, "-01:00")));
+  EXPECT_EQ(1, monthTimestampWithTimezone(TimestampWithTimezone(0, "+00:00")));
   EXPECT_EQ(
       11,
-      evaluateWithTimestampWithTimezone<int64_t>(
-          "month(c0)", 123456789000, "+14:00"));
+      monthTimestampWithTimezone(
+          TimestampWithTimezone(123456789000, "+14:00")));
   EXPECT_EQ(
       2,
-      evaluateWithTimestampWithTimezone<int64_t>(
-          "month(c0)", -123456789000, "+03:00"));
+      monthTimestampWithTimezone(
+          TimestampWithTimezone(-123456789000, "+03:00")));
   EXPECT_EQ(
       4,
-      evaluateWithTimestampWithTimezone<int64_t>(
-          "month(c0)", 987654321000, "-07:00"));
+      monthTimestampWithTimezone(
+          TimestampWithTimezone(987654321000, "-07:00")));
   EXPECT_EQ(
       9,
-      evaluateWithTimestampWithTimezone<int64_t>(
-          "month(c0)", -987654321000, "-13:00"));
+      monthTimestampWithTimezone(
+          TimestampWithTimezone(-987654321000, "-13:00")));
   EXPECT_EQ(
       std::nullopt,
-      evaluateWithTimestampWithTimezone<int64_t>(
```

Comment:
Do we still have other callsites for `evaluateWithTimestampWithTimezone()`? I suppose the plan is to remove it at some point?

Reply:
Yes, there are still other call sites that need to be removed. I am gonna be removing them in smaller incremental chunks to make the code review process easier.